scripts/bin/create.py

```diff
@@ -191,6 +190,14 @@ def move_files_to_dist(dirs, src, dst):
         shutil.move(full_path, dst)
 
 
+def add_repository_type(repo, repository_type):
+    filename = "%s/repository.yaml" % repo
+    if os.path.exists(filename):
+        with open(filename, "a") as f:
+            f.write("\ntype: %s" % repository_type)
+        f.close()
+
+
 def clone_repo(repo, github_token):
     command = 'git clone https://%s@github.com/hmrc/%s' % (github_token, repo)
     print(f"cloning: {command}")
```

Comment:
nothing to do with your change but does this line basically print out the github token given its part of the url?

Reply:
good spot - yes it does. Will remove